MonoGame.Framework/Media/MediaPlayer.WMS.cs

```diff
@@ -231,6 +240,12 @@ private static void OnTopologyReady()
             _volumeController = CppObject.FromPointer<AudioStreamVolume>(volumeObjectPtr);
 
             SetChannelVolumes();
+
+            if (_desiredPosition.HasValue)
+            {
+                StartSession(_desiredPosition.Value);
+                _desiredPosition = null;
+            }
```

Comment:
Man, this Media Foundation library is so finicky! It doesn't work properly for me either if I try to specify the position when first starting the session. It kinda sucks that this needs to be done, but  this workaround does seem to work for me (on my Win7 machine at least).

However, I'd recommend adding a comment to the code to explain why this is being done otherwise it won't make sense to other people looking at this later.